src/assistant/ResultMenu/SuggestionItemTextSecondary.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * Code copied and adapted from cozy-drive
+ *
+ * See source: https://github.com/cozy/cozy-drive/blob/fbe2df67199683b23a40f476ccdacb00ee027459/src/modules/search/components/SuggestionItemTextSecondary.jsx
+ */
+import React from 'react'
+
+import AppLinker from 'cozy-ui/transpiled/react/AppLinker'
+import SuggestionItemTextHighlighted from './SuggestionItemTextHighlighted'
+import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
+import Link from 'cozy-ui/transpiled/react/Link'
+
+const SuggestionItemTextSecondary = ({ text, query, url, slug }) => {
+  const { isMobile } = useBreakpoints()
+
+  if (isMobile || !url) {
```

Comment:
Should we secure this by checking also the value of `slug` in addition to `url`?

Reply:
I don't think so, because I tested with an `undefined` slug, and it still works :shrug: 
This does not look mandatory, but AppLinker complains if it is missing, so I kept it